ui/nano.html

```diff
@@ -299,24 +299,15 @@
           </div>
           `,
         created: function() {
-          try {
-            const clientPlugin = window.Grid.PluginHost.getPluginByName(
-              this.client.name
-            )
-            this.listenForStateUpdates(clientPlugin)
-          } catch (error) {
-            console.log('error polling for state updates', error)
-          }
+          this.clientPlugin = this.plugins.find(p => p.name === this.client.name)
```

Comment:
getting console errors for every plugin: `Cannot read property 'find' of undefined`